ouroboros-network-api/src/Ouroboros/Network/NodeToClient/Version.hs

```diff
@@ -36,10 +36,10 @@ data NodeToClientVersion
     | NodeToClientV_14
     -- ^ added @GetPoolDistr@, @GetPoolState@, @GetSnapshots@
     | NodeToClientV_15
+    -- ^ added `query` to NodeToClientVersionData
```

Comment:
Should we also add the `GetStakeDelegDeposits` query that @lehins added?

Reply:
Hmm, this would definitely be thing to do to keep things consistent with how they are now; but now that Network and Consensus live in different repos, it could also be argued that we should refrain from mentioning Consensus queries here, and rather have a place in Consensus where this is mentioned. We could talk about this in the Consensus Office Hours tomorrow.